src/Alaouy/Youtube/Youtube.php

```diff
@@ -248,19 +248,34 @@ public function getChannelById($id, $optionalParams = false, $part = ['id', 'sni
      * @return array
      * @throws \Exception
      */
-    public function getPlaylistsByChannelId($channelId, $optionalParams = array(), $part = ['id', 'snippet', 'status'])
+    public function getPlaylistsByChannelId($channelId, $pageToken = false, $maxResults = 50, $optionalParams = array(), $part = ['id', 'snippet', 'status'])
     {
         $API_URL = $this->getApi('playlists.list');
         $params = array(
             'channelId' => $channelId,
             'part' => implode(', ', $part),
+            'maxResults' => $maxResults
         );
+
         if ($optionalParams) {
             $params = array_merge($params, $optionalParams);
         }
+
+        // Pass page token if it is given, an empty string won't change the api response
+        if (is_string($pageToken)) {
+            $params['pageToken'] = $pageToken;
+        }
+
         $apiData = $this->api_get($API_URL, $params);
 
-        return $this->decodeList($apiData);
+        $result = ['results' => $this->decodeList($apiData)];
```

Comment:
@Rpsl Without the need for the extra parameters, this would do the trick here 
```php
$result = ['results' => $this->decodeList($apiData)];

$result['info']['nextPageToken'] = (isset($this->page_info['nextPageToken']) ? $this->page_info['nextPageToken'] : false);
$result['info']['prevPageToken'] = (isset($this->page_info['prevPageToken']) ? $this->page_info['prevPageToken'] : false);
```

Please update your PR and docs accordingly and I will merge it ;)